src/main/java/embl/ebi/variation/eva/pipeline/steps/writers/VariantAnnotationWriter.java

```diff
@@ -0,0 +1,17 @@
+package embl.ebi.variation.eva.pipeline.steps.writers;
+
+import embl.ebi.variation.eva.pipeline.MongoDBHelper;
+import embl.ebi.variation.eva.pipeline.annotation.load.VariantAnnotationMongoItemWriter;
+import org.opencb.datastore.core.ObjectMap;
+
+/**
+ * TODO this class is the translation of the original code. This class can be removed lated if parent class can be refactored.
```

Comment:
Which refactoring would it require? I guess it mainly has to do with the method `writeVariantAnnotationInMongoDb`?